akka-docs/src/main/paradox/scala/fault-tolerance.md

```diff
@@ -31,27 +31,43 @@ in more depth.
 
 For the sake of demonstration let us consider the following strategy:
 
-@@snip [FaultHandlingDocSpec.scala]($code$/scala/docs/actor/FaultHandlingDocSpec.scala) { #strategy }
+Scala
+:  @@snip [FaultHandlingDocSpec.scala]($code$/scala/docs/actor/FaultHandlingDocSpec.scala) { #strategy }
 
-I have chosen a few well-known exception types in order to demonstrate the
+Java
+:  @@snip [FaultHandlingTest.java]($code$/java/jdocs/actor/FaultHandlingTest.java) { #strategy }
+
+We have chosen a few well-known exception types in order to demonstrate the
 application of the fault handling directives described in @ref:[supervision](general/supervision.md).
 First off, it is a one-for-one strategy, meaning that each child is treated
 separately (an all-for-one strategy works very similarly, the only difference
 is that any decision is applied to all children of the supervisor, not only the
-failing one). There are limits set on the restart frequency, namely maximum 10
-restarts per minute; each of these settings could be left out, which means
-that the respective limit does not apply, leaving the possibility to specify an
-absolute upper limit on the restarts or to make the restarts work infinitely.
-The child actor is stopped if the limit is exceeded.
+failing one). 
+In the above example, `10` and @scala[`1 minute`]@java[`Duration.create("1 minute")`] are passed to the `maxNrOfRetries`
```

Comment:
for Java I'd use `Duration.create("1, TimeUnit.MINUTES")` instead of the fragile string parsing